### PR TITLE
Add async await in test FormSubmitter for ru translate

### PR DESCRIPTION
**Pull request by airdry**

### CODE REVIEW

File: src/ru/simulating-user-input.md [hunk 1]
```diff
@@ -8,26 +8,23 @@
...
-      <input type="submit">
+      <input v-model="username" data-username />
+      <input type="submit" />
     </form>
 
-    <div 
-      class="message" 
-      v-if="submitted"
-    >
+    <div class="message" v-if="submitted">
```
Comment (lmiller1990) [inline src/ru/simulating-user-input.md]: can we keep these the same as the demo component? Eg, no need to change them.

File: src/ru/simulating-user-input.md [hunk 2]
```diff
@@ -64,14 +61,16 @@ import { shallowMount } from "@vue/test-utils"
 import FormSubmitter from "@/components/FormSubmitter.vue"
 
 describe("FormSubmitter", () => {
-  it("Показывает сообщение после отправки", () => {
+  it("Показывает сообщение после отправки", async () => {
```
Comment (lmiller1990) [inline src/ru/simulating-user-input.md]: Thanks

**Comment by airdry** [comment]
Done

**Comment by lmiller1990** [comment]
great thanks!

**Pull request merged**